app/server/lib/Sharing.ts

```diff
@@ -336,7 +337,9 @@ export class Sharing {
       }
       await this._activeDoc.processActionBundle(ownActionBundle);
 
-      const actionSummary = await this._activeDoc.handleTriggers(localActionBundle);
+      const actionSummary = !isCalculate ?
```

Comment:
`isCalculate` has become misleading and should probably be renamed as it includes other actions. I don't think it should be used here, the exception should probably exist only for the actual Calculate action. In particular it would make sense to trigger webhooks after a RespondToRequests.

Reply:
Thanks for the hint!